avro-data/src/main/java/io/confluent/connect/avro/AvroData.java

```diff
@@ -1600,7 +1600,7 @@ private Object toConnectData(Schema schema, Object value, ToConnectContext toCon
             }
             if (converted == null) {
               throw new DataException(
-                  "Did not find matching union field for data: " + value.toString());
+                  "Did not find matching union field for data: <redacted>");
```

Comment:
The reason I kept was to avoid accidental re-introduction of values in future in case someone wants to add them for troubleshooting/debugging. This would make it clear that we cannot print this.
Let me know if you still feel this should be removed.

Reply:
Since the customer sees this, I would rather not print <redacted>.  Perhaps add a comment instead?

Reply:
Done